assets/js/src/core/modules/app/app-loader.tsx

```diff
@@ -119,17 +122,29 @@ export const AppLoader = (props: IAppLoaderProps): React.JSX.Element => {
       })
   }
 
+  const loadUserData = (): void => {
+    Promise.all([
+      initSettings(),
+      initActivePerspective()
```

Comment:
We still have a problem here. `initSettings`and `initActivePerspective` need to be executed in sequential order as the perspective logic uses a setting from the settings endpoint. Otherwise depending on the timing we can get this error:

![image](https://github.com/user-attachments/assets/08820389-fb53-4dec-8e17-b00f25f74d9c)

Could you please add a logic so that we first load the settings and then the perspectives? Please prepare in a way so that we can also load other endpoints in parallel to the perspectives but after loading the settings.